src/boot/ual.js

```diff
@@ -15,9 +18,135 @@ export default boot(async ({ app, store }) => {
     ]
   };
 
+  async function loginHandler() {
+    let accountName = 'eosio';
+    let permission = 'active';
+    if (localStorage.getItem('autoLogin') === 'cleos') {
+      accountName = localStorage.getItem('account');
+    } else {
+      await new Promise((resolve) => {
+        Dialog.create({
+          class: 'cleos-auth-dialog',
+          color: 'primary',
+          title: store.state.account.justViewer ? 'Navigate as viewer' : 'Connect to cleos',
+          message: 'Account name',
+          prompt: {
+            model: '',
+            type: 'text',
+          },
+          cancel: true,
+          persistent: true,
+        })
+        .onOk((data) => {
+            accountName = data !== '' ? data : 'eosio';
+        })
+        .onCancel(() => {
+            throw 'Cancelled!';
+        })
+        .onDismiss(() => {
+            resolve(true);
+        });
+      });
+      if (store.state.account.justViewer) {
+        permission = "active";
+      } else {
+        await new Promise((resolve) => {
+          Dialog.create({
+            class: 'cleos-auth-dialog',
+            color: 'primary',
+            title: 'Connect to cleos',
+            message: 'Account permission',
+            options: {
+              type: 'radio',
+              model: [],
+              items: [
+                { label: 'Active', value: 'active' },
```

Comment:
I understand that by using `cleos` we need to specify which account authorization we're using, but I wonder if this is an essential step for the end user. 

This issue aims to provide a friendly way for non-technical users to see other people's transactions, so I'm wondering if we could hide this selection from the user when using this feature. 🤔

Reply:
"to provide a friendly way for non-technical users to see other people's transactions" if that's the only goal, then there's no point, it's already available via account view in explorer. probably worth revisiting.

Reply:
This PR brings two ways of using Cleos authenticator. The normal was adding it to the list of authenticators with a custom couple of handlers (for login and for signing).

The second and the requested way was adding a special button on the home, near the normal login button, which only asks for the name of the account (not for the authorization which is always assumed as 'active'.

Take a look at the title of the dialogs, they read differently depending on the button you used.

"Navigating as viewer" vs "connect to cleos"
![image](https://user-images.githubusercontent.com/4420760/207967720-1a35ab76-0337-4815-a5f4-6602193b14f2.png)

Reply:
My bad, I still had the last changes locally in my PC and didn't push them. Now I did and you can see what I mean:

[bugfix: actually differentiating the two ways of cleos login](https://github.com/telosnetwork/telos-wallet/pull/89/commits/f9a6f3ca504753695cebb4300d8df4b197f59583)